lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5001,26 +5002,14 @@ void SharedEmitterState::emitOps(EmissionList &thingsToEmit, raw_ostream &os,
   }
 }
 
-/// Prepare the given MLIR module for emission.
```

Comment:
Is there a reason to drop this? Looks like it'd be useful for two copies of its body below (modulo the `LoweringOptions options(module)` first).